assessments/projects/cortex/self-assessment.md

```diff
@@ -0,0 +1,259 @@
+# Cortex - Self-assessment
+
+The Self-assessment is the initial document for Cortex to begin thinking about the security of the project, determining gaps in its security, and preparing any security documentation for their users.
+
+**Authors:** Dhanraj Chavan @heydc7 Raiya Haque @raiyahaque Abdul Alhazmi @Abkzmii Sushanth Rabipalli sushanth3120
+
+**Contributors/Reviewers:** Pranava Kumar Vemula (@Rana-KV)
+
+## Self-assessment outline
+
+### Table of contents
+
+* [Metadata](#metadata)
+  * [Security links](#security-links)
+* [Overview](#overview)
+  * [Actors](#actors)
+  * [Actions](#actions)
+  * [Background](#background)
+  * [Goals](#goals)
+  * [Non-goals](#non-goals)
+* [Self-assessment use](#self-assessment-use)
+* [Security functions and features](#security-functions-and-features)
+* [Project compliance](#project-compliance)
+* [Secure development practices](#secure-development-practices)
+* [Security issue resolution](#security-issue-resolution)
+* [Appendix](#appendix)
+* [Action Items](#action-items)
+
+## Metadata
+
+|   |  |
+| -- | -- |
+| Software | https://github.com/cortexproject/cortex |
+| Security Provider | No |
+| Languages | Go |
+| SBOM | `Not Applicable` |
```

Comment:
```suggestion
| SBOM | https://github.com/cortexproject/cortex/releases/latest/download/sbom.tar.gz |
```

Reply:
While Cortex isn't automatically generating the SBOM at build time (ideal), they do have a policy they appear to be following to manually generate it at release time using Kubernetes BOM.

https://github.com/cortexproject/cortex/blob/master/RELEASE.md#sign-the-release-artifacts-and-generate-sbom

Reply:
OK Will add.